src-conf/mssql.sql

```diff
@@ -1327,12 +1327,12 @@ create table workflow_task (
    mod_date datetime null,
    due_date datetime null,
    created_by NVARCHAR(255) null,
-   assigned_to NVARCHAR(255) null,
+   assigned_to NVARCHAR(36) null,
    belongs_to NVARCHAR(255) null,
    title NVARCHAR(255) null,
    description NVARCHAR(MAX) null,
-   status NVARCHAR(255) null,
-   webasset NVARCHAR(255) null,
+   status NVARCHAR(36) null,
+   webasset NVARCHAR(36) null,
```

Comment:
If the size of these fields is changed here, shouldn't it be changed in all other scripts as well (Oracle, Postgres, etc.)? Is this something specific to a customer's DB?